acctest/main_test.go

```diff
@@ -173,7 +190,7 @@ func TestShouldFailOnTlsVerificationWithSelfSignedCert(t *testing.T) {
 	// then
 	assert.Equal(t, 127, exitcode)
 	assert.Empty(t, stdout)
-	assert.Contains(t, stderr, "rpc error")
+	assert.Contains(t, stderr, "TLS handshake")
```

Comment:
Well, if we log `TLS handshake` upon issue with TLS verification it's already good, but it's still hard to guess how exactly the error message looks like from this test.

Reply:
On one hand, I didn't want to specify exact message in test, on the other, having test with exact wording really helps to understand, what is the expected behavior.

I'm leaning towards updating all tests to verify exact messages. If no objections to that, will commit shortly.

Reply:
Well, I'm not sure if the whole message is really required but at least a portion of it that describes the error. Details like path to the cert or its fingerprint are less interesting from the test perspective, IMO. But either way is OK with me.

Reply:
just to clarify, we can't distinct from grpc api between connection refused and TLS handshake failed. So the message will be same for both.

Reply:
Maybe it worth including this remark into the error message so that the end user will be aware of detail?